### PR TITLE
Dev created /solutions/ai/genai

### PR DESCRIPTION
**Pull request by immortalcodes**
## Done

Dev created /solutions/ai/genai

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check for all things related to addition of new page.
- [Design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=873-6710&node-type=instance&m=dev)
- [Copydoc](https://docs.google.com/document/d/1eMvLlW6U-3ZWRRtY3Bq2u6EaFRqop4n6x8wgj3OWw-k/edit?tab=t.0)
- [Demo](https://canonical-com-1402.demos.haus/solutions/ai/genai)

## Issue / Card

Fixes # [WD-15327](https://warthogs.atlassian.net/browse/WD-15327)

## Screenshots

[if relevant, include a screenshot]

[WD-15327]: https://warthogs.atlassian.net/browse/WD-15327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/solutions/ai/genai.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>
+          Learn more
+          <br class="u-hide--medium" />
```
Comment (britneywwc) [inline templates/solutions/ai/genai.html:177]: The breakpoint doesn't match with design, should be after GenAI

**Review by britneywwc — COMMENTED** [review]
Looks good, just a couple minor things.

The dynamic form modal's country field doesn't have the dropdown selection
Screenshot 2024-10-23 at 12 20 04 PM

**Review by britneywwc — APPROVED** [review]
Thanks for making the changes, beautiful work! 👯

**Comment by Sophie-32** [comment]
For navigation, this should follow the bubble navigation: https://docs.google.com/document/d/1Y0dxbuar0UAPSCfILtP7fsEX-83yDh7Ha637FA9Wbik/edit?tab=t.0

File: secondary-navigation.yaml [hunk 2]
```diff
@@ -125,3 +125,13 @@ telco:
       path: /solutions/telco/5g-edge
     - title: Open RAN
       path: /solutions/telco/open-ran
+
+ai:
+  title: AI
+  path: /solutions/ai
+
+  children:
+    - title: AI
```
Comment (akbarkz) [inline secondary-navigation.yaml:134]: according to the copy doc this item should be for Edge AI

**Review by akbarkz — CHANGES_REQUESTED** [review]

**Review by akbarkz — APPROVED** [review]

**Pull request merged**